crates/swc/benches/isolated_declarations.rs

```diff
@@ -0,0 +1,50 @@
+use std::{path::Path, sync::Arc};
+
+use criterion::{criterion_group, criterion_main, Criterion};
+use swc_common::{comments::SingleThreadedComments, FilePathMapping, Mark, SourceMap, GLOBALS};
+use swc_ecma_parser::{parse_file_as_program, Syntax};
+use swc_ecma_transforms::{fixer::paren_remover, resolver};
+use swc_typescript::fast_dts::{FastDts, FastDtsOptions};
+
+fn mk() -> swc::Compiler {
+    let cm = Arc::new(SourceMap::new(FilePathMapping::empty()));
+    swc::Compiler::new(cm)
+}
+
+fn bench_isolated_declarations(criterion: &mut Criterion) {
+    criterion.bench_function("isolated-declarations", |b| {
+        let c = mk();
+        let fm =
+            c.cm.load_file(Path::new("benches/assets/vue-id.ts"))
+                .unwrap();
+        b.iter_with_large_drop(|| {
```

Comment:
`iter_with_large_drop` is effective only if you return the variable with large drop.

Reply:
Yes, I find there is no difference if I change it to `iter`.